android-lib/build.gradle

```diff
@@ -76,6 +76,7 @@ configurations {
 
 buildscript {
   ext.kotlin_version = '1.3.41'
+  ext.androidx_version = '1.3.0-alpha02'
```

Comment:
@heyzooi @b-stolyarov Do you have any concerns about using an `alpha` version of `androidx`?

Reply:
right, better to change to the latest stable version, which is `1.2.0`